src/test/ui/lint/unused/issue-103320-must-use-ops.rs

```diff
@@ -0,0 +1,30 @@
+// check-pass
+
+#![warn(unused_must_use)]
+#![feature(never_type)]
+
+use std::ops::Add;
+use std::ops::Sub;
+use std::ops::Mul;
+use std::ops::Div;
+use std::ops::Rem;
+
+fn main() {
+    let x = 2_u32;
+    x.add(4); //~ WARN unused return value of `add` that must be used
+    //~^ WARN unused arithmetic operation that must be used
```

Comment:
It's unfortunate to have these doubled warnings, but I think that's because you added both the `must_use` message and the match in `unused.rs`. That is, we get output for both `fn_warned` and `op_warned`. Can we combine them somehow, or give preference to one?

Reply:
Yes，great catch, thanks!